src/AbstractSQLRules2SQL.ts

```diff
@@ -77,6 +77,8 @@ const UnknownValue: MetaMatchFn = (args, indent) => {
 		case 'Bind':
 		case 'Cast':
 		case 'Coalesce':
+		case 'JSONPath':
```

Comment:
You've got a few (leftover?) `JSONPath` references around the place with no handler for them

Reply:
Yeah, that was a mistake while doing some debugging. It's now set as a TextValue and I've removed it from the UnknownValue list.